.teamcity/settings.kts

```diff
@@ -79,17 +79,18 @@ object Build_Release: BuildType({
         param("env.NEXT_PUBLIC_FEEDBACK_API", "https://alphasite.dts-stn.com/api/feedback")
         param("env.NEXT_CONTENT_API", "%vault:dts-secrets-dev/data/digitalCentre!/NEXT_CONTENT_API%")
         param("env.NEXT_PUBLIC_ADOBE_ANALYTICS_URL", "%vault:dts-secrets-dev/data/digitalCentre!/NEXT_PUBLIC_ADOBE_ANALYTICS_URL%")
-        param("env.OCP_APIM_SUBSCRIPTION_KEY", "%vault:dts-secrets-dev/data/digitalCentre!/OCP_APIM_SUBSCRIPTION_KEY%")
-        param("env.CPP_ACTIVE_BENEFIT_URL", "%vault:dts-secrets-dev/data/digitalCentre!/CPP_ACTIVE_BENEFIT_URL%")
-        param("env.EI_ACTIVE_BENEFIT_URL", "%vault:dts-secrets-dev/data/digitalCentre!/EI_ACTIVE_BENEFIT_URL%")
-        param("env.TARGET", "dev")
+        param("env.OCP_APIM_SUBSCRIPTION_KEY", "%vault:dts-secrets-dev/digitalCentre!/OCP_APIM_SUBSCRIPTION_KEY%")
+        param("env.CPP_ACTIVE_BENEFIT_URL", "%vault:dts-secrets-dev/digitalCentre!/CPP_ACTIVE_BENEFIT_URL%")
+        param("env.EI_ACTIVE_BENEFIT_URL", "%vault:dts-secrets-dev/digitalCentre!/EI_ACTIVE_BENEFIT_URL%")
+        param("env.APP_URL", "https://sc-digital-centre-dashboard.bdm-dev.dts-stn.com:443")
```

Comment:
It seems there are a few of these `app_urls` that don't match the branch/target in this file.

Reply:
same thing like the above discussion

Reply:
This one is different - while it would be nice to be dynamic, it at least needs to match the build it's targeting; here the url should be `https://sc-digital-centre-release.bdm-dev.dts-stn.com:443` if it has to be hard coded.